include/sys/spa.h

```diff
@@ -142,12 +143,6 @@ typedef struct dva {
 	uint64_t	dva_word[2];
```

Comment:
Wow, I hope this is a github rendering issue, otherwise these are some exciting characters.